docs/tools/util-linux.md

```diff
@@ -6,24 +6,24 @@ sidebar_label: Linux System Utilities
 
 # Linux System Utilities
 
-As defined by the project itself, *util-linux* is a random collection of
-Linux&reg; utilities. This project is hosted
-on <a href="https://github.com/karelzak/util-linux" target="_blank">GitHub</a>.
-This project generally packaged in most distributions under the name
-*util-linux* and installed by default.
+*util-linux* is, as the documentation of the *util-linux* project itself says, 
+a random collection of Linux&reg; utilities. The *util-linux* project is hosted
+<a href="https://github.com/karelzak/util-linux" target="_blank">here on GitHub</a> or <a href="https://github.com/util-linux/util-linux" target=_blank">here on Github</a>. This project is packaged in most distributions under the name
+*util-linux* and is installed by default.
```

Comment:
Why the line break so early ?

Reply:
In this case, I thought that the long paragraph was really three paragraphs:

(1) What util-linux is.
(2) What util-linux does.
(3) What util-linux is particularly good for.

If you think that this should be squashed into the same long paragraph, I'll defer to you. I think breaking up information into paragraphs like this makes it easier for people to digest the information in them.

Reply:
I think it is fine as is. I was merely remarking that the lines:

project is packaged in most distributions under the name
*util-linux* and is installed by default.

Could be better packed, filling up to 80 char limit:

project is packaged in most distributions under the name *util-linux* and is 
installed by default.

Not a big deal :)